test/unit/ngsiv2/contextRequests/ngsildPayloadMeasure.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "Second MQTT Device",
+    "type": "AnMQTTDevice",
+    "status": {
+        "type": "Property",
+        "value": "free"
+    },
+    "category": {
+        "type": "Property",
+        "value": [
+            "onstreet"
+        ]
+    },
+    "refParkingSite": {
+        "type": "Relationship",
+        "value": "urn:ngsi-ld:ParkingSite:santander:daoiz_velarde_1_5"
+    },
```

Comment:
Note that, for the original value represented in NGSI-LD, relationship has `object` property instead of `value`. NTC